Mathlib/RingTheory/FiniteType.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johan Commelin
 -/
 import Mathlib.RingTheory.Adjoin.Tower
+import Mathlib.RingTheory.Ideal.QuotientOperations
```

Comment:
Is `Mathlib/RingTheory/Ideal/Quotient.lean` not enough?

Reply:
No. It requires the instance `Algebra R (S ⧸ I)`.

Reply:
Okay fair enough.